src/addons/blog/services/blog-sync.ts

```diff
@@ -95,8 +95,8 @@ import { AddonBlogOfflineEntryDBRecord } from './database/blog';
      * @returns Syncronization result.
      */
     async performEntriesSync(siteId: string): Promise<AddonBlogSyncResult> {
-        const result: AddonBlogSyncResult = { updated: false, warnings: [] };
         const entriesToSync = await this.syncEntriesToRemove(siteId);
+        const result: AddonBlogSyncResult = entriesToSync.result;
```

Comment:
```suggestion
        const result = entriesToSync.result;
```
This cast is no longer needed, now the type can be inferred. You can also do this:

`        const { entries, result } = await this.syncEntriesToRemove(siteId);`